diff --git a/sway/handlers.c b/sway/handlers.c
index 3abe2fc..ad6c1c1 100644
--- a/sway/handlers.c
+++ b/sway/handlers.c
@@ -807,6 +807,10 @@ static bool swayc_border_check(swayc_t *c, const void *_origin) {
 	const struct wlc_point *origin = _origin;
 	const struct wlc_geometry title_bar = c->title_bar_geometry;
 
+	if (c->border_type != B_NORMAL) {
+		return false;
+	}
+
 	if (origin->x >= title_bar.origin.x && origin->y >= title_bar.origin.y
 		&& origin->x < title_bar.origin.x + (int32_t)title_bar.size.w
 		&& origin->y < title_bar.origin.y + (int32_t)title_bar.size.h) {
