diff -ur sway-0.10-rc1.orig/sway/focus.c sway-0.10-rc1/sway/focus.c
--- sway-0.10-rc1.orig/sway/focus.c	2016-10-01 14:57:40.372096540 +0100
+++ sway-0.10-rc1/sway/focus.c	2016-10-01 14:58:07.864451336 +0100
@@ -114,7 +114,7 @@
 		c = get_focused_container(workspace);
 	}
 
-	swayc_log(L_DEBUG, c, "Setting focus to %p:%lu" PRIuPTR, c, c->handle);
+	swayc_log(L_DEBUG, c, "Setting focus to %p:%" PRIuPTR, c, c->handle);
 
 	if (c->type == C_VIEW) {
 		// dispatch a window event
